From 874dadf1168f8a1b2b1bd4ab5bb4a20097147ab0 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 28 May 2020 08:01:37 -0700
Subject: [PATCH] pd-mapper: Include limits.h for PATH_MAX

Fixes
pd-mapper.c:199:22: error: 'PATH_MAX' undeclared (first use in this function); did you mean 'AF_MAX'?

Upstream-Status: Submitted [https://github.com/andersson/pd-mapper/pull/4]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 pd-mapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pd-mapper.c b/pd-mapper.c
index 45177ae..f5d45ee 100644
--- a/pd-mapper.c
+++ b/pd-mapper.c
@@ -36,6 +36,7 @@
 #include <fcntl.h>
 #include <libgen.h>
 #include <libqrtr.h>
+#include <limits.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-- 
2.26.2

